listenbrainz/db/tests/test_feedback.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+import listenbrainz.db.feedback as db_feedback
+import listenbrainz.db.user as db_user
+
+from listenbrainz.db.testing import DatabaseTestCase
+
+
+class FeedbackDatabaseTestCase(DatabaseTestCase):
+
+    def setUp(self):
+        DatabaseTestCase.setUp(self)
+        self.user = db_user.get_or_create(1, 'recording_feedback_user')
+
+    def insert_test_data(self, user_id, neg_score=False):
+        """ Insert test data into the database """
+
+        if neg_score:
+            # Insert the two records with negative score
+            with open(self.path_to_data_file('recording_feedback_negative_score.json')) as f:
```

Comment:
I don't think we need a seperate json file for these tests, I'd prefer if you just kept the dict in code, it's easier to read that way.

Reply:
Removed.